mdsuite/calculators/einstein_diffusion_coefficients.py

```diff
@@ -112,6 +121,7 @@ def __call__(
         atom_selection: np.s_ = np.s_[:],
         molecules: bool = False,
         tau_values: Union[int, List, Any] = np.s_[:],
+        fit_range: int = -1,
```

Comment:
I would vote for `None` over `-1`

Reply:
The -1 here was sort of motivated by the fact that array[-1] is the last element which is where it is taken.

Reply:
Okay, then I like the `-1`. But maybe the name `range` is a bit misleading then? So this defines the stop/max value of the range argument? Maybe name it `fit_max` / `fit_stop` or so? But given that `for x in range(stop)` automatically is the stop value we can also keep it.

Reply:
Yeah I also struggled with the name here, it isn't a range without the starting point it is just a stop index. fit max index or somethign maybe?

Reply:
Don't we have someone in our team who gave a talk on variable naming @christophlohrmann ?

Reply:
fit_max_index is better than range imo. It also tells that an int is expected (unlike tau_values)

Reply:
sounds good to me.